deployment/cloud/aws/infrastructure/modules/server/userdata

```diff
@@ -41,9 +41,11 @@ runcmd:
   # install pixi
   - echo "Installing Pixi"
   - export PIXI_HOME=/opt/pixi
+  - export HOME=/root
   - curl -fsSL https://pixi.sh/install.sh | bash
   - export PATH=$PIXI_HOME/bin:$PATH
   - ln -s $PIXI_HOME/bin/pixi /usr/local/bin/pixi
+  - unset HOME
```

Comment:
and then thought "hmm, i wonder if this was unset for a reason? lemme just leave the environment the way i found it for the rest of the script"